### DISCUSSION TITLE
Fastapi root_path ignored while using api versioning

### DISCUSSION THREAD

**Discussion by Rajesh2015** [fastapi/fastapi#8505] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from mangum import Mangum
import os
from fastapi.middleware.cors import CORSMiddleware
from fastapi_versioning import VersionedFastAPI

stage = os.environ.get('STAGE', None)
openapi_prefix = f"/{stage}" if stage else "/"
app = FastAPI(title="MyAwesomeApp", root_path="/dev/")


@app.get("/")
def get_root():
    return {"message": "FastAPI running in a Lambda function"}

@app.get("/info")
def get_root():
    return {"message": "TestInfo"}
app=VersionedFastAPI(app, enable_latest=True,
                               description='MyAwesomeApp')
app.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)
handler = Mangum(app)
```

### Description

- Open browser and call end point /docs. In network tab we can see it's trying to fetch from  [](https://url/openapi.json )with versioning enabled which is unexpected
- Without versioning it try to call [](https://url/dev/openapi.json) which is correct in terms of behaviour 
Due to above issue it's difficult to open the doc page when deployed in aws api gateway 
https://github.com/tiangolo/fastapi/issues/4955

### Operating System

Linux

### FastAPI Version

0.70.1

### Python Version

3.8.10

**Comment by j-carson** [comment, marked as answer]
I think you’re on the wrong issues forum here. The library you’re having a problem with is a different code base.  https://github.com/DeanWay/fastapi-versioning
